recipes/lenskit/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "lenskit" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f2da93c7144cc048edbac0c34578f4a150a2949f0bcc5e16de5eef59dd64d31a
+
+build:
+  number: 0
+  script: "{{ PYTHON }} setup.py install"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - pytest-runner
+    - pip
+    - numba >=0.43
+    - pandas >=0.24
+    - numpy >=1.16
+    - scipy >=1.2
+    - pyarrow >=0.15
+    - cffi >=1.12.2
+    - binpickle >=0.3.2
+    - mkl-devel  # to build helper library
```

Comment:
Are most of these actually needed to compile?

Reply:
They are in the source package's `install_requires`, so the package installation will try to find them if they aren't already installed.

Reply:
host is different than install thoug

Reply:
`install_requires` is equivalent to conda's `run` (`host` is `setup_requires`)

Reply:
Ok. Moved.